pkg/app/test_command.go

```diff
@@ -36,6 +38,10 @@ func TestCommand(testPath string, ctx TestCommandContext) error {
 		fmt.Println("Starting test against directory: " + testPath + "...")
```

Comment:
What happens if `ctx.Dir` is true and the path is an URL?

Reply:
`files, err := ioutil.ReadDir(directory)` will respond with 
```
commander (fetch-test-suites-over-http) $ ./commander test --dir https://raw.githubusercontent.com/commander-cli/commander/master/examples/commander.yaml
Starting test against directory: https://raw.githubusercontent.com/commander-cli/commander/master/examples/commander.yaml...

open https://raw.githubusercontent.com/commander-cli/commander/master/examples/commander.yaml: no such file or directory
```
and 

```
./commander test --dir examples/commander.yaml Starting test against directory: examples/commander.yaml...

Hello
fdopendir: not a directory
```
I think a better error here would be 
```
commander (fetch-test-suites-over-http) $ ./commander test --dir example/ 
Starting test against directory: example/...

Error: Input is not a directory
commander (fetch-test-suites-over-http) $ ./commander test --dir https://raw.githubusercontent.com/commander-cli/commander/master/examples/commander.yaml
Starting test against directory: https://raw.githubusercontent.com/commander-cli/commander/master/examples/commander.yaml...

Error: Input is not a directory
commander (fetch-test-suites-over-http) $ ./commander test --dir examples/commander.yaml Starting test against directory: examples/commander.yaml...

Error: Input is not a directory
commander (fetch-test-suites-over-http) $ ./commander test --dir -
Starting test against directory: -...

Error: Input is not a directory
```

Reply:
That looks good, thank you!